cmd/podman/main.go

```diff
@@ -43,12 +43,18 @@ func main() {
 	if filepath.Base(os.Args[0]) == registry.PodmanSh ||
 		(len(os.Args[0]) > 0 && filepath.Base(os.Args[0][1:]) == registry.PodmanSh) {
 		shell := strings.TrimPrefix(os.Args[0], "-")
+		cfg := registry.PodmanConfig()
 
-		args := []string{shell, "exec", "-i", "--wait", strconv.FormatUint(uint64(registry.PodmanConfig().ContainersConfDefaultsRO.Engine.PodmanshTimeout), 10)}
+		if err := cfg.ContainersConfDefaultsRO.Engine.Validate(); err != nil {
```

Comment:
Is this necessary because of the containers.conf changes you made?

Reply:
Yes indeed.

It's not strictly necessary but because those parameters are being transferred directly and unchecked to the `args` slice, I thought a `Validate` method that throws human-friendly errors might be helpful to have.

Reply:
I think the whole `Validate` method can be omitted by using the `omitempty` toml flags. Will update it accordingly.

Reply:
Updated accordingly.